src/components/OpenSeadragonViewer.js

```diff
@@ -209,8 +209,8 @@ export class OpenSeadragonViewer extends Component {
           ref={this.ref}
         >
           { children }
+          <ZoomControls windowId={windowId} />
         </div>
-        <ZoomControls windowId={windowId} />
       </>
```

Comment:
Should we also remove the `<></>`?

Reply:
/shrug no idea. It might be nice to leave incase we need to drop anything else in there in the future and reduce the diff? 

I could go either way

Reply:
WFM.